src/components/Header.jsx

```diff
@@ -44,12 +44,12 @@ function Header() {
     <Container>
       {alert}
       <Grid container className={classes.header} spacing={3}>
-        <Grid item>
+        <Grid item sm={8}>
           <MuiLink component={Link} to="/" variant="h4" underline="none">
             {BRAND_NAME.toUpperCase()}
           </MuiLink>
         </Grid>
-        <Grid item>
+        <Grid item sm={4}>
```

Comment:
This fixed the glitch that the avatar gets shifted down in the header, it now remains to the right of the window until a certain screen width when it begins to shift down from the container parent

Reply:
I believe this was part of #352 . Please leave this change there and remove it from this PR.

Reply:
ok